docs/snippets/common/storybook-auto-docs-override-mdx-container.js.mdx

```diff
@@ -0,0 +1,20 @@
+```js
+// CustomPageProvider.js
+
+import { MDXProvider } from '@mdx-js/react';
+
+import { DocsContainer } from '@storybook/blocks';
+
+import * as DesignSystem from 'your-design-system';
+
+export const MyDocsContainer = (props) => (
+  <MDXProvider
+    components={{
+      h1: DesignSystem.H1,
+      h2: DesignSystem.H2,
+    }}
+  >
+    <DocsContainer {...props} />
+  </MDXProvider>
+);
+```
```

Comment:
Probably good to not assume they read the prior section and just show how to apply it here too:

```suggestion
export const parameters = {
  actions: { argTypesRegex: '^on[A-Z].*' },
  controls: {
    matchers: {
      color: /(background|color)$/i,
      date: /Date$/,
    },
  },
  docs: {
    container: MyDocsContainer,
  },
};
```
```